docs/adrs/9999-use-playwright.md

```diff
@@ -0,0 +1,80 @@
+
+## Considered Alternatives
+
+- Continue using existing version of Cypress
+- Upgrade to current version of Cypress
+- Convert integration tests to Playwright, gradually
+- **Convert integration tests to Playwright, all at once**
+
+## Decision Outcome
+
+First, we've decided that the current situation with using a long out-of-date version of Cypress is untenable. While it still maintains some functionality -- it has caught legitimate errors that in pull requests would have otherwise been merged into the `main` branch of the code base -- the existing level of flakiness is not something we should accept. Additionally, there are a broad range of developers on the project touching this code, and very little concept of standards or best practices; it can reasonably be assumed that the rate of sporadic and consistent test failures will increase, as will the rate of false negative tests that don't actually cover what we believe it does. Finally, being stuck on an outmoded piece of code makes us functionally incapable of any security fixes; the longer we go in this state, the more painful any forced update will eventually be.
+
+Give that, if we are going to need to re-write a significant portion (if not all) of our integration test code base, it makes sense at this juncture to evaluate other tools that may be a better fit for our application and needs. [Playwright](https://playwright.dev/) is a well-supported integration test library that's seeing rapid community adoption.
+
+Comparable to Cypress in aim, Playwright has the benefit of improved tooling around automatic test generation (increasing developer velocity), as well as much stronger, interactive reporting of errors as opposed to Cypress' video output. Unlike Cypress, the interface of code written for Playwright is similar to that of React Testing Library, which we're already using for our front end unit tests. The compile-time and runtime for Playwright tests, especially locally, is also improved over the Cypress baseline, providing a significantly shorter feedback loop that allows for better developer iteration.
+
+One final consideration is around the timeframe of changing over to any new tool. We have multiple efforts nominally underway, but functionally stalled, that would gradually let us change our way of working from one library or technique to another. We want to avoid another iteration of this pattern, and while we are cognizant of the risks and communication costs of any sort of cutover, we will seek to convert all existing Cypress-based integration tests to Playwright at once, immediately removing our dependence on the older tool.
```

Comment:
I think it would be good to write about how we did an experiment and converted some tests from Cypress to Playwright and found that we could do so pretty quickly. That gives us some data to believe that converting to Playwright will be at most the same as fixing Cypress and at best, faster.

Thoughts?